### PR TITLE
Bend

### PR DESCRIPTION
**Pull request by RohanVashisht1234**

### CODE REVIEW

File: plugins/language_bend.lua [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+-- mod-version:3
+
+local syntax = require "core.syntax"
+
+syntax.add {
+    name = "Bend",
+    files = { "%.bend$" },
+    comment = "#",
+    patterns = {
+        { pattern = { "#", "\n" },                type = "comment" },
```
Comment (Guldoman) [inline plugins/language_bend.lua:12]:
```suggestion
        { pattern = "#.*",                        type = "comment" },
```

**Review by Guldoman — COMMENTED** [review]
Needs a version bump of `meta_languages`.

**Comment by RohanVashisht1234** [comment]
Ok gaman, I have increased the version.

**Comment by RohanVashisht1234** [comment]
Ok, gman, I have updated the version.

**Pull request merged**